CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 2.0.0 - [#24](https://github.com/openfisca/country-template/pull/24)
+
+* Technical change
+* Details:
+  - Introduce the use of a keywords to reference enum item.
+  - EnumCols (e.g. housing_occupancy_status), now reference an Enum Class (e.g. HousingOccupancyStatus) that has a name & description for each item and an explicit default value.
+
+Remove:
```

Comment:
The Changelog of a package is mainly targeted to _re-users_. When reading a block as a user, I should get answers to the questions : 

- Am I impacted by this change ? 
- What new feature can I use ?
- What feature I used to use is now deprecated / remove ? What should I use instead ? 
- How has the _public API_ of the package has changed ? Did the format of the input / output change ?

From the country-template's user perspective, the fact that the syntax of enums changed is not significant. It matters for the _maintener_ of this package, but the _public API_  is not impacted.

However, the _public API_ is impacted by the fact that, when setting an input enum variable, I am now supposed to set `free_lodger`, and can't use `2` or `Free lodger` anymore.

The same goes for the output : when requesting the calculation of an enum variable, I should now be expecting `free_lodger`, and not `Free lodger`.

These two changes impact me if I use the Web API, or the python API, or when I write YAML tests. These are the _real breaking changes_ that need to be documented in this changelog 🙂 .